src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -5,115 +5,141 @@
     }
 
     /**
      * Check whether specified value is even
+     *
      * @param value to check
      * @return true if value is even, false - otherwise
      */
     public static boolean isEven(int value) {
-        //TODO: implement me
-        return false;
+        return value % 2 == 0;
     }
 
     /**
      * Check whether specified value is odd
+     *
      * @param value to check
      * @return true if value is odd, false - otherwise
      */
     public static boolean isOdd(int value) {
-        //TODO: implement me
-        return false;
+        return value % 2 != 0;
     }
 
     /**
      * Check whether specified value is prime
+     *
      * @param value to check
      * @return true if value is prime, false - otherwise
      */
     public static boolean isPrime(int value) {
-        //TODO: implement me
-        return false;
+        if (value < 2)
+            return false;
+        for (int i = 2; i * i <= value; i++)
+            if (value % i == 0)
+                return false;
+        return true;
     }
 
     /**
      * Parse specified char array and create instance of {@code ExtendedInteger}
+     *
      * @param value to parse
      * @return instance of {@code ExtendedInteger} or
      * null in case specified value is null or the value does not contain a parsable integer
      */
     public static ExtendedInteger parseInt(char[] value) {
-        //TODO: implement me
+        if (value == null || value.length == 0)
         return null;
+        String newValue = new String(value);
+        return ExtendedInteger.parseInt(newValue);
     }
 
     /**
      * Parse specified string and create instance of {@code ExtendedInteger}
+     *
      * @param value to parse
      * @return instance of {@code ExtendedInteger} or
      * null in case specified value is null or the value does not contain a parsable integer
      */
     public static ExtendedInteger parseInt(String value) {
-        //TODO: implement me
+        if (value == null | value.length() == 0)
         return null;
+        StringBuilder sb = new StringBuilder();
+        if (Character.isDigit(value.charAt(0)) || value.startsWith("-"))
+            sb = sb.append(value.charAt(0));
+        for (int i = 1; i < value.length(); i++) {
+            if (Character.isDigit(value.charAt(i))) {
+                sb = sb.append(value.charAt(i));
+            } else return null;
+        }
+        return new ExtendedInteger(new Integer(value));
+
     }
 
     /**
      * Get int representation of {@code ExtendedInteger}
+     *
      * @return int representation
      */
     public int getValue() {
-        //TODO: implement me
-        return 0;
+        return value;
     }
 
     /**
      * Check whether current value is even
+     *
      * @return true if value is even, false - otherwise
      */
     public boolean isEven() {
-        //TODO: implement me
-        return false;
+        return this.value % 2 == 0;
```

Comment:
As you can see, you have code duplication: there is no need to implement same method 2 times, as you can just call static method from non-static.
Also, assume, that you'll need to add some condition to this implementation, for example: isEven should return true if value is equal to `673` - you'll need to change your implementation 2 times: first one in static method and second one in non-static method.

Same comment about `isOdd` and `isPrime` implementations.